javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -496,21 +497,20 @@ export async function generateBootnodeSpec(
 }
 
 interface UsedNames {
-  [properyName: string]: number;
+  [properyName: string]: boolean;
 }
 
 const mUsedNames: UsedNames = {};
 
 export function getUniqueName(name: string): string {
-  let uniqueName;
-  if (!mUsedNames[name]) {
-    mUsedNames[name] = 1;
-    uniqueName = name;
-  } else {
-    uniqueName = `${name}-${mUsedNames[name]}`;
-    mUsedNames[name] += 1;
+  let uniqueName = `${name}-${generateNamespace()}`;
```

Comment:
This was causing naming collisions right? There is another issue with the cleanup but using some random here sounds good :)